### PR TITLE
Team Black and more

### PR DESCRIPTION
**Pull request by odeimaiz**
## What do these changes do?
- [x] Nodes tree action buttons context dependant ITISFoundation/osparc-issues#261
  - Add-new-node removed
  - Delete and Open buttons react to selection change

#### Bonus
- [x] Do not create study from template if user has no access to all services
- [x] Bugfix: Reopen study when reloading page. Also fixes some issues with "Study is already open"

## Related issue number
closes ITISFoundation/osparc-issues#261
closes ITISFoundation/osparc-issues#275

### CODE REVIEW

File: services/web/client/source/class/osparc/store/Store.js [hunk 1]
```diff
@@ -221,7 +221,7 @@ qx.Class.define("osparc.store.Store", {
           .then(study => {
             osparc.data.Resources.fetch("studies", "state", params)
               .then(state => {
-                study["locked"] = state["locked"];
+                study["state"] = state;
```
Comment (ignapas) [inline services/web/client/source/class/osparc/store/Store.js:224]:
```suggestion
                study.state = state;
```

File: services/web/client/source/class/osparc/store/Store.js [hunk 2]
```diff
@@ -268,7 +268,7 @@ qx.Class.define("osparc.store.Store", {
               .then(states => {
                 states.forEach((state, idx) => {
                   const study = studies[idx];
-                  study["locked"] = state["locked"];
+                  study["state"] = state;
```
Comment (ignapas) [inline services/web/client/source/class/osparc/store/Store.js:271]:
```suggestion
                  study.state = state;
```

**Review by ignapas — COMMENTED** [review]
little bug: if selection hasn't change, like for example after reloading the page, the state of the buttons can be incorrect
image

**Review by pcrespov — APPROVED** [review]

**Review by ignapas — CHANGES_REQUESTED** [review]
Found a problem when opening an iteration:
Kapture 2020-08-26 at 17 23 37
And direct selection of nodes in workbench does not update tree buttons:
Kapture 2020-08-26 at 17 26 29

**Review by ignapas — APPROVED** [review]
Cool! Only a minor: the sweeper window doesn't adapt to content, for example when you want to go back to the main study. It keeps the previous size from when it was holding the iteration generator.
image

**Pull request merged**